cedar-lean/Cedar/Thm/Data/Map.lean

```diff
@@ -54,76 +169,274 @@ theorem not_contains_of_empty {α β} [BEq α] (k : α) :
 theorem make_wf [LT α] [StrictLT α] [DecidableLT α] (xs : List (α × β)) :
   WellFormed (Map.make xs)
 := by
-  simp only [WellFormed, make, toList, kvs, List.canonicalize_idempotent]
+  simp [WellFormed, make, toList, kvs, List.canonicalize_idempotent]
 
-theorem make_mem_list_mem [LT α] [StrictLT α] [DecidableLT α] {xs : List (α × β)} :
```

Comment:
Let's leave the old signatures here.  

If a new theorem is necessary or helpful, it can be added as well, and it can use the old one in the proof.

Reply:
I think it's fine for the two theorems to have asymmetric names. But if we want symmetry, value operations / theorems can be wordier versions of the key operations' names, something like `make_mem_values_mem`.